src/lib/resource/sections/applicant-fatal-claim.js

```diff
@@ -13,13 +13,13 @@ module.exports = {
                     title: 'Select why you are applying',
                     oneOf: [
                         {
-                            title: 'I am applying because of an injury',
+                            title: 'I am applying because of an injury.',
```

Comment:
Full stops have been added to all of these properties but titles and descriptions in other pages on this PR do not?

Reply:
The files in the PR follow these rules: Titles have no full stops unless question marks are needed or wanted. Descriptions have full stops. Error messages have full stops only if they have more than one sentence.